truco-front/src/gameLogic/GameActionsManager.ts

```diff
@@ -65,6 +66,9 @@ export default class GameActionsManager {
     }
 
     public handleAction(actionMessage: GameActionMessage, iCalled: boolean = false) {
+        if(!iCalled && actionMessage.action !== GameAction.PLACE_CARD && actionMessage.action !== GameAction.NONE){
+            toast(getPrintableAction(actionMessage.action))

```

Comment:
This line of code is part of the view, this logic shouldn't be here.

Reply:
ready